tests/functional/args.test.ts

```diff
@@ -101,3 +102,31 @@ Deno.test("wut args", async test => {
     assertEquals(runTest(["+tea.xyz/magic", "env"]), "dump")
   })
 })
+
+Deno.test("reads env", () => {
+  const keys = ["CI", "CLICOLOR", "CLICOLOR_FORCE", "DEBUG", "GITHUB_ACTIONS", "GITHUB_TOKEN",
```

Comment:
probs should add a commend in the file where we declare these. Potentially can use typescript to generate the array